doc/compiler.md

```diff
@@ -55,9 +55,7 @@ riot.compile(function() {
 
 Compilation phase is basically free and takes no time at all. Compiling a [timer tag](https://github.com/riot/riot/blob/master/test/tag/timer.tag) 30 times takes 2 milliseconds on a regular laptop. If you have a crazy page with 1000 different timer-sized tags, the compilation takes around 35ms.
 
-The compiler weights only 3.2KB (1.7K gzipped) so you can safely perform client side compilation on production without download or performance or issues.
-
-Just like Riot itself the compiler works on IE8 as well.
+The compiler weights so you can safely perform client side compilation on production without download or performance or issues.
```

Comment:
'The compiler is lightweight' instead of 'The compiler weights' (that should have been weighs)